incubator/insights-sumologic/README.md

```diff
@@ -0,0 +1,150 @@
+          fieldpass = [ "raw" ]
+          data_format = "carbon2" 
+```
+
+### Sumo Logic Dashboards
+
+---
+
+1. [Sumo Logic Collectors](#sumo-logic-collectors) for NuoDB Insight data.
+
+2. [NuoDB Insights graphs](#insights-graphs-on-sumo-logic-dashboards) in Sumo Logic dashboards.
+
+3. [Kubernetes Graphs](#sumo-logic-kubernetes-graphs) in Sumo Logic (Optional)
+
+#### Sumo Logic Collectors
+
+---
+
+Website is https://www.sumologic.com
+
+Create a new account or use an existing account. For a new account, skip past the introductions and go straight to the dashboard. At the time of writing, the skip link was below the introduction text.
+
+Click on <b>Manage Data</b>: 
+
+![Manage Data](./images/sumologic0.png)
+
+Click on **Add Collector**
+
+![Add Collector](./images/sumologic1.png)
+
+Select **Hosted Collector**
+
+![Hosted Collector](./images/sumologic2.png)
+
+Add an appropriate name for the collector.
+
+Next is to add a source to the newly created collector. At the time of writing, the source was under "Generic" and is  "HTTP logs and metrics"
+
+![Generic_logs and metrics](./images/sumologic4.png)
+
+Add an appropriate name for the source for the collector. 
+
+Next is to copy and keep safe the Sumo Logic metrics endpoint. This will be needed in the section for the helm database values file setup. It will be something like:
+
+```
+https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/ZaVnC4dhaV0laeqZ1_FuM8b_P9gPRX4DIJzrJ8eaIy0Lz0rQH5o-bIARKiKf7M09suw6VFJpKSIt_3zv0hq80kcq0VivLeZWM3yXey7EXDG5YFFBaMWqMg==```
+```
+
+![sumologic endpoint](./images/sumologic6.png)
+
+#### 
+
+#### Insights graphs on Sumo Logic dashboards
+
+---
+
+Click on Personal -> three dots to the right of the side bar and create a new folder.
+
+![Personal_Folder](./images/sumologic21.png)
+
+Click on the 3 dots to the right of the folder name and select Import
+
+![Import](./images/sumologic22.png)
+
+Upload the NuoDB dashboard JSON as a copy paste operation.  The JSON files that can be copy pasted can be found in the following locations
+
+```
+files/NuoDB_AdHoc.json
+files/NuoDB_DBA.json
+files/NuoDB_NOC.json
+```
+
+Click "Import" at the bottom of the pane.
+
+ ![Nuodb_Dashboard_JSON](./images/sumologic25.png)
+
+Set the Collector name and Source name to be the same as set in section 1.
+
+![Set_Collector_and_Source](./images/sumologic26.png)
+
+**Note:** The Sumo Logic NuoDB graphs are a subset of the graphs that are available in Insights. Data that is sent to Sumo Logic is a subset of all the data that is sent to Insights. 
+
+#### Sumo Logic Kubernetes Graphs
+
+---
+
+From the App Catalog search for classic apps "kubernetes" -> "Kubernetes" and follow the install instructions. 
+
+**Tip:** Use a different namespace other than the namespace for the NuoDB install. 
+
+**Tip:** Take care to make a note of the ```accessId```, ```accessKey```, ```clusterName``` and ```collectorName```.  ```accessKey``` is shown only once.
```

Comment:
I could only see the collector URL in the screenshots but not the rest. Are these pre-generated keys so that you can login to Sumologic console?

Reply:
Yes, pregenerated keys. When installing the Sumo Logic helm chart they are generated and displayed on the screen with install instructions. They look something like below. Should I include these instructions?

```
helm -n sl upgrade --install sumologic sumologic/sumologic \
  --namespace=sl \
  --create-namespace \
-f - <<EOF
sumologic:
  accessId: suHgE6N1d1itvp
  accessKey: tw8wRIywOEch9dAjqXRRKJ7IGjd0tR7HI2kHS2pf3jEDhDxCwSHa9QQ0wucN76gT
  clusterName: Kubernetes_cluster-2024-08-02T14:24:07.871Z
  collectorName: kubernetes-2024-08-02T14:24:07.872Z
  setup:
    monitors:
      enabled: false
EOF
```

Reply:
It probably makes sense to include the instructions, redacting the key info. Added.